assembly/src/library/mod.rs

```diff
@@ -97,6 +97,14 @@ impl Module {
         }
     }
 
+    // PUBLIC ACCESSORS
+    // --------------------------------------------------------------------------------------------
+
+    /// Returns a reference to the ast.
+    pub fn ast(&self) -> &ModuleAst {
+        &self.ast
+    }
+
```

Comment:
We don't need this accessor as `ast` is already a public field (on line 80 above).